### PR TITLE
deploy node & gpu exporter

### PR DESCRIPTION
**Pull request by YanjieGao**
deploy node exporter and gpu exporter in one same pod.

### CODE REVIEW

File: service-deployment/bootstrap/prometheus/start.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+pushd $(dirname "$0") > /dev/null
+
+NAMESPACE=${NAMESPACE:-monitor}
+KUBECTL="kubectl  --namespace=\"${NAMESPACE}\""
+eval "kubectl create namespace \"${NAMESPACE}\""
+
+eval "${KUBECTL} create -f node-exporter-ds.yaml"
+eval "${KUBECTL} get pods"
```
Comment (fanyangCS) [inline service-deployment/bootstrap/prometheus/start.sh]: don't need this. if you really want to make sure it is started. please refer to [https://github.com/Microsoft/pai/blob/master/service-deployment/kubernetesTool/servicestatus.py](https://github.com/Microsoft/pai/blob/master/service-deployment/kubernetesTool/servicestatus.py)

**Comment by fanyangCS** [comment]
> COPY start.sh /usr/local/start.sh

after copy, pls chmod +x to make sure it can execute

---
Refers to: service-deployment/src/gpu-exporter/dockerfile:33 in 002cb96. [](commit_id = 002cb96c2af94799189b144ea9272422a1ad50a9, deletion_comment = False)

File: service-deployment/bootstrap/prometheus/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+NAMESPACE=${NAMESPACE:-monitor}
+KUBECTL="kubectl  --namespace=\"${NAMESPACE}\""
+
+INSTANCES="daemonset/node-exporter"
+
+for instance in ${INSTANCES}; do
+  eval "${KUBECTL} delete --ignore-not-found --now \"${instance}\""
+done
+
+eval "${KUBECTL} get pods"
```
Comment (ydye) [inline service-deployment/bootstrap/prometheus/cleanup.sh]: remove this line

File: service-deployment/bootstrap/prometheus/start.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+popd > /dev/null
```
Comment (fanyangCS) [inline service-deployment/bootstrap/prometheus/start.sh]: don't remove this line. a pushd need a popd

**Pull request merged**